dev-vcs/gitqlient/gitqlient-1.6.2.recipe

```diff
@@ -75,34 +61,29 @@ BUILD_PREREQUIRES="
 	cmd:qmake$secondaryArchSuffix >= 5
 	"
 
-defineDebugInfoPackage gitqlient$secondaryArchSuffix \
```

Comment:
Any reason to remove?

Reply:
Not a good one, lol. I removed it when I was troubleshooting packaging errors and I felt like it wasn't entirely necessary. Fixed.